### PR TITLE
add anyscale

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
needs license

**Comment by timkpaine** [comment]
waiting on proper license packaging, need someone from anyscale to do this @krfricke @ericl

**Comment by timkpaine** [comment]
@conda-forge/help-python all set, note that the license is non-standard

File: recipes/anyscale/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - pip
+  commands:
+    - pip check
+    - anyscale --version
+
+about:
+  home: https://docs.anyscale.com/reference/cli
+  summary: Command Line Interface for Anyscale
+  description: Command Line Interface for Anyscale
+  license: Anyscale-License
```
Comment (ocefpaf) [inline recipes/anyscale/meta.yaml:69]: Let's use the SPDX format for custom licenses and the acronym in the license file.

```suggestion
  license: LicenseRef-AS
```

**Comment by xhochy** [comment]
This packages a `test` folder/module. This should be removed. 

@conda-forge/core Can someone please also check the license? I would not be comfortable approving it: https://www.anyscale.com/as-license

> General. Your downloading or use of the Software will be deemed your approval of these terms (“AS-License”).  You acknowledge and agree that Anyscale may update this AS-License from time to time at any time by posting an updated version of these terms on [anyscale.com/as-license](https://anyscale.com/as-license) and that your continued use of the Software following an updated version being posted will constitute consent to such changes and such use shall be subject to the current published version of the AS-License. If you do not agree to these AS-License terms, you may not view or make any use of the Software.

The aspect of the license changing independently of the downloaded source and "may not view" doesn't sound like something we can host here.

**Comment by ocefpaf** [comment]
Argh! Good catch! I don't like that part and missed it in the first pass. Sorry.

**Pull request merged**